task-manager/src/routers/task.js

```diff
@@ -0,0 +1,83 @@
+const express = require('express');
+const router = new express.Router();
+const Task = require('../models/task');
+
+router.post('/tasks', async (req, res) => {
+    const task = new Task(req.body);
+
+    try {
+        await task.save();
+        res.status(201).send(task);
+    } catch (error) {
+        res.status(400).send(error);
+    }
+});
+
+router.get('/tasks', async (req, res) => {
+    try {
+        const tasks = await Task.find({});
+        res.send(tasks);
+    } catch (error) {
+        res.status(500).send(error);
+    }
+});
+
+router.get('/tasks/:id', async (req, res) => {
+    const _id = req.params.id;
```

Comment:
También puedes definir esto como
```suggestion
    const { id } = req.params;
```

Reply:
Es que esta definido asi porque luego en las funciones como `findById`, estas reciben como parametro `_id`, entonces para hacer uso del shorthand notation. Si uso destructuring pues me tocaria poner `_id: id`, entonces por evitar eso y ya jaja.